### DISCUSSION TITLE
Issues with text color on rofi-1.7.0

### DISCUSSION THREAD

**Discussion by ikbenlike** [davatorium/rofi#1462] [(Possible) Issues] [answered]
After the recent update I tried to convert my Xresources theme to a rasi file, [which can be found here](https://gist.github.com/ikbenlike/73695dfda5429ceace8233208df215dd).

this is what the Xresources looked like:
```
rofi.color-enabled: true
rofi.color-window: #212121, #71c2af, #212121
rofi.color-normal: #71c2af, #212121, #71c2af, #212121, #71c2af
rofi.color-active: #a6a663, #212121, #a6a663, #212121, #a6a663
rofi.color-urgent: #a66363, #212121, #a66363, #212121, #a66363
```

when I run rofi with the theme by doing `rofi -show run -theme cyan` it looks like this:
![image](https://user-images.githubusercontent.com/8676901/133948115-8c683ffc-353a-43a1-aa25-304fb3f830c1.png)

while I would want/expect it to look more like this (screenshot is from the old theme generator):
![image](https://user-images.githubusercontent.com/8676901/133948230-b5c940af-0336-4d64-ab96-65bac2fa4e18.png)

I seem to be unable to get the text-color of a selected normal element to change: it seems to default to rgb(0,0,0), both when I use something I previously declared, and when I use a hex-formatted colour code in-place. I am running this on arch linux with bspwm, if that matters.

**Comment by DaveDavenport** [comment, marked as answer]
you theme is incorrect., It does not theme the correct widgets.

Try adding this:
```
element-text, element-icon {
    background-color: inherit;
    text-color:       inherit;
}
```

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#basic-structure-1

**Reply by ikbenlike** [reply]
Thanks, that does exactly what I need. I'm not able to figure out how to color the text in the inputbar, however, and I am not sure what I would need to change that.

**Reply by DaveDavenport** [reply]
I think this should do it:
```css
entry {
text-color: pink;
}
```

See the above ASCII art.

You can test this by:
```bash
 rofi -show run -theme-str 'entry { text-color: pink; }'
```
